service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -600,6 +614,80 @@ void shouldNotifyAdminOfPlacementApplicationUpload() throws Exception {
         );
     }
 
+    @Test
+    void shouldSendNotificationToPartiesWhenAddedToCaseByEmail() throws IOException, NotificationClientException {
+        final Map<String, Object> expectedParameters = ImmutableMap.<String, Object>builder()
+            .put("firstRespondentLastName", "Moley")
+            .put("familyManCaseNumber", "123")
+            .put("reference", "12345")
+            .build();
+
+        List<Element<Representative>> representatives = new ArrayList<>();
+        Element<Representative> representative = Element.<Representative>builder()
+            .value(Representative.builder()
+                .email("joe-blogs@gmail.com")
+                .servingPreferences(EMAIL).build()).build();
+        representatives.add(representative);
+
+        RepresentativeServingPreferences preferences = EMAIL;
+
+        given(partyAddedToCaseContentProvider.getPartyAddedToCaseNotificationParameters(callbackRequest().getCaseDetails(),
+            preferences)).willReturn(expectedParameters);
+
+        given(partyAddedToCaseContentProvider.getPartyAddedToCaseNotificationTemplate(preferences))
+            .willReturn(PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE);
+
+        notificationHandler.sendNotificationToPartiesAddedToCase(new PartyAddedToCaseEvent(callbackRequest(), AUTH_TOKEN, USER_ID, representatives));
+
+        verify(notificationClient, times(1)).sendEmail(
+            eq(PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE), eq(PARTY_ADDED_TO_CASE_BY_EMAIL_ADDRESS),
+            eq(expectedParameters), eq("12345"));
+    }
+
+    @Test
+    void shouldSendNotificationToPartiesWhenAddedToCaseThroughDigitalService() throws IOException, NotificationClientException {
+        final Map<String, Object> expectedParameters = ImmutableMap.<String, Object>builder()
+            .put("firstRespondentLastName", "Moley")
+            .put("familyManCaseNumber", "123")
+            .put("reference", "12345")
+            .build();
+
+        List<Element<Representative>> representatives = new ArrayList<>();
+        Element<Representative> representative = Element.<Representative>builder()
+            .value(Representative.builder()
+                .email("damian@swansea.gov.uk")
+                .servingPreferences(DIGITAL_SERVICE).build()).build();
+        representatives.add(representative);
+
+        RepresentativeServingPreferences preferences = DIGITAL_SERVICE;
```

Comment:
This can be inlined

Reply:
This was just being pushed up so resolving

Reply:
previously resolved